src/appengine/handlers/jobs.py

```diff
@@ -44,6 +53,30 @@ def get_queues():
   return queues
 
 
+def get_results(this):
+  """Get results for the jobs page."""
+  params = dict(this.request.iterparams())
+  # Return jobs sorted alphabetically by name
```

Comment:
nit: add an empty line before a comment line

Reply:
actually it would be even better to move line 58 (`params = ...`) to line 62, as `params` isn't used anywhere before that

Reply:
done